examples/Notebook.ipynb

```diff
@@ -33,17 +33,32 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "doc = GISDocument()"
+    "doc = GISDocument(\"france_hiking.jGIS\")"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "a0314dc6-9d79-4361-8b86-60b84f543cf9",
+   "id": "5938ba70-36d7-4af3-b877-815464ef6d1a",
    "metadata": {},
    "outputs": [],
    "source": [
-    "doc"
+    "doc.add_raster_layer(\n",
+    "    url=\"https://mt1.google.com/vt/lyrs=y&x={x}&y={y}&z={z}\",\n",
+    "    name=\"Google Satellite\",\n",
+    "    attribution=\"Google\",\n",
+    "    opacity=0.6\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "69ac2a9a-d9a9-4170-b1f5-451b9fd5780e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "doc.add_geojson_layer(path=\"examples/france_regions.json\")"
```

Comment:
```suggestion
    "doc.add_geojson_layer(path=\"france_regions.json\")"
```

Reply:
It didn't seem to trow an error if the file was not found.

Reply:
This would require fixing https://github.com/QuantStack/jupytergis/issues/39

Reply:
> It didn't seem to trow an error if the file was not found

Yes because it does not verify the file existence Python side, I will push a change for that.

Reply:
It seems that my suggestion still has to be applied?

Reply:
That's why I did not resolve yet the comment :)